tasks/main.yml

```diff
@@ -51,8 +51,6 @@
   template:
     src: gridinit.conf.j2
     dest: "{{ openio_gridinit_config_file }}"
-  with_items: "{{ openio_gridinit_services }}"
```

Comment:
The commit message does not explain that, I'd put:
`Always install the global gridinit config file, even when there's no 'present' service to setup`
Or something like that, as this also explains the purpose of the the modification